distributed/worker.py

```diff
@@ -2103,6 +2131,40 @@ def story(self, *keys):
                            for c in msg
                            if isinstance(c, (tuple, list, set)))]
 
+    @property
+    def client(self):
+        """ Get local client attached to this worker
+
+        If no such client exists, create one
+
+        See Also
+        --------
+        get_client
+        """
+        with self._lock:
+            if self._client:
+                return self._client
+
+            try:
+                from .client import default_client
+                client = default_client()
+            except Exception:
```

Comment:
It would be nice to know why/when this fails. `except Exception` is a bit vague (and might hide programming errors?).